blog/content/post/2020-04-27-trenchboot-nlnet-release-04.md

```diff
@@ -0,0 +1,305 @@
+
+    ```
+    $ wget -O tb-minimal-image.wic.bmap https://cloud.3mdeb.com/index.php/s/3c5QNHbNRx5gpY5/download
+    $ wget -O tb-minimal-image.wic.gz https://cloud.3mdeb.com/index.php/s/xd9z3iDS3gkPrmQ/download
+    ```
+
+3. Using `bmaptool` flash SSD disk with downloaded image.
+
+    > **IMPORTANT**: Make sure to use proper target device - /dev/sdX. In our
+    case it is /dev/sda, but can be different for you.
+
+    Usage of bmaptool is: `bmaptool copy --bmap <image.wic.bmap> <image.wic.gz> </dev/sdX>`
+
+    ```
+    $ bmaptool copy --bmap tb-minimal-image.wic.bmap tb-minimal-image.wic.gz /dev/sda
+    [ 1076.049347]  sda: sda1 sda2
+    bmaptool: info: block map format version 2.0
+    bmaptool: info: 532480 blocks of size 4096 (2.0 GiB), mapped 48007 blocks (187.5 MiB or 9.0%)
+    bmaptool: info: copying image 'tb-minimal-image.wic.gz' to block device '/dev/sda' using bmap file 'tb-minimal-image.wic.bmap'
+    (...)
+    bmaptool: info: 99% copied
+    [ 1120.645490]  sda: sda1 sda2
+    ```
+
+4. Reboot platform and boot from just flashed SSD disk.
+
+    You should see GRUB menu with 2 entries:
+
+    - boot - it is 'normal' boot without DRTM
+    - secure-boot - it is boot with **DRTM enabled**
+
+5. Choose `secure-boot` entry, verify bootlog and enjoy platform with DRTM!
+
+    At the beginning, there should be similar output:
+
+    ```
+    grub_cmd_slaunch:122: check for manufacturer
+    grub_cmd_slaunch:126: check for cpuid
+    grub_cmd_slaunch:136: set slaunch
+    grub_cmd_slaunch_module:156: check argc
+    grub_cmd_slaunch_module:161: check relocator
+    grub_cmd_slaunch_module:170: open file
+    grub_cmd_slaunch_module:175: get size
+    grub_cmd_slaunch_module:180: allocate memory
+    grub_cmd_slaunch_module:192: addr: 0x100000
+    grub_cmd_slaunch_module:194: target: 0x100000
+    grub_cmd_slaunch_module:196: add module
+    grub_cmd_slaunch_module:205: read file
+    grub_cmd_slaunch_module:215: close file
+    grub_slaunch_boot_skinit:41: real_mode_target: 0x8b000
+    grub_slaunch_boot_skinit:42: prot_mode_target: 0x1000000
+    grub_slaunch_boot_skinit:43: params: 0xcfe2391
+    code32_start 0x0000000001000000:
+    (...)
+    ```
+
+    > It indicates that DRTM is enabled and executed. For details about bootflow
+    and DRTM verification we refer to [previous article](https://blog.3mdeb.com/2020/2020-04-03-trenchboot-nlnet-lz-validation/)
+    in which it is precisely described.
+
+## CI/CD system
+
+As it is mentioned in the beginning of this post, we have introduced **CI/CD
+system** to build each TrenchBoot components. It is a big step towards
+development and validation automation. Also it is a convenient way to deliver
+all necessary up-to-date binaries. This section describes how this environment
+is built, what tools do we use and how You can utilize this system.
+
+### Continuous Integration / Continuous Delivery
+
+
+### Our CI/CD system
+
+
+### Theory of operation
```

Comment:
@piotr-kleins usage of artifacts should also be documented

Reply:
Entire CI/CD section is now written. Artifacts are also described